src/main/java/com/nutomic/syncthingandroid/activities/MainActivity.java

```diff
@@ -49,12 +52,15 @@
     @Override
     @SuppressLint("InflateParams")
     public void onApiChange(SyncthingService.State currentState) {
+        if (mGui == null) {
```

Comment:
`Activity#isDestroyed()` should do the trick ;)

(and I'd put it next to `if isFinishing()` below)